.github/dependabot.yml

```
@@ -8,6 +8,18 @@ updates:
       - "PR: waiting for review"
       - "PR: dependencies"
     open-pull-requests-limit: 15
+    groups:
+      babel:
+        patterns:
+          - "@babel/*"
+          - "babel-*"
+      eslint:
+        patterns:
+          - "eslint"
+          - "eslint-*"
+      fortawesome:
+        patterns:
+          - "@fortawesome/*"
```

Small comment so that we don't think this is a typo in the future:
```suggestion
        # This is not a typo in this file, the packages are really called fortawesome, even though the project is called fontawesome.
        patterns:
          - "@fortawesome/*"
```

---

That due to their organization name

https://github.com/FortAwesome

So I wonder if this comment is needed since the `@` prefix is for author/organization name not package name

---

Email address is `admin@fortawesome.com`
https://fortawesome.com links back to GH organization

From their [Twitter](https://twitter.com/getfortawesome)
> Design, build, optimize, and serve your front end assets faster! From the makers of 
[@fontawesome](https://twitter.com/fontawesome)